runtime/gear/src/lib.rs

```diff
@@ -442,6 +545,8 @@ construct_runtime!(
         TransactionPayment: pallet_transaction_payment = 6,
         Session: pallet_session = 7,
         Utility: pallet_utility = 8,
+        Proxy: pallet_proxy = 9,
```

Comment:
pls set 10 12 so that it is possible to add the necessary pallets between them.

Reply:
As discussed offline, leaving the discriminants as is for the meantime as they don't seem to ever require reordering.